src/admin/components/UserResumeTable/index.tsx

```diff
@@ -13,16 +13,40 @@ interface UserResumeTableProps {
 
 const UserResumeTable: React.FC<UserResumeTableProps> = (props) => {
   const { resumes } = props;
+  const [yearFilters, setYearFilters] = useState([{ text: '2026', value: '2026' }]);
   const [selectedRowKeys, setSelectedRowKeys] = useState<React.Key[]>([]);
 
+  const updateYearFilters = () => {
+    setYearFilters(
+      resumes
+        .filter((value: UserResume, index: number) => {
+          // Resumes.indexOf(value) === index checks for first occurance
+          return resumes.indexOf(value) === index;
+        })
+        .sort()
+        .map((element) => ({ text: `${element.user.graduationYear}`, value: `${element.user.graduationYear}` })),
```

Comment:
Can we format this object to be more readable? (Start keys on a new line)
```suggestion
        .map((element) => ({ 
            text: `${element.user.graduationYear}`, 
            value: `${element.user.graduationYear}` 
        })),
```